interfaces/Portalicious/src/app/services/registrations-table-column.service.ts

```diff
@@ -30,15 +30,15 @@ import {
 } from '~/services/registration-attribute.service';
 import { TranslatableStringService } from '~/services/translatable-string.service';
 
-export const FILTERABLE_ATTRIBUTES_LABELS: Record<string, string> = {
+const FILTERABLE_ATTRIBUTES_LABELS: Record<string, string> = {
```

Comment:
I thought some ESLint-rule was already taking care of these "unused variables"...

Good to know it doesn't. ;)